workers/data-url.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<title>data URL worker</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id=log></div>
+<script>
+var worker = new Worker("data:,fetch('/').then(() => self.postMessage('fail'), () => self.postMessage('pass'))") // not same-origin
+worker.onmessage = (e) => {
+  assert_equals(e.data, "pass")
```

Comment:
testharness.js does implicitly create an `async_test` here, but for some reason a failure is still a harness error. @jgraham, how do these implicitly created tests work?